onnxruntime/core/providers/openvino/contexts.h

```diff
@@ -79,6 +80,7 @@ struct ProviderInfo {
   uint32_t num_of_threads{0};              // [num_of_threads]: Overrides the accelerator default value of
                                            // number of threads with this value at runtime.
   config_t load_config{};                  // JSON config map to load custom OV parameters.
+  shape_t shape{};                         // Used for reshaping ov tensors to a particular lower and upper bound
```

Comment:
would it be possible to name this variable as the EP-specific option named - "reshape_input", or at least mention that name in the comment?
As this is the only place where OVEP-specific options are listed in comprehensible format, so we're referring to these names/comments.

Could also the description of this be added to https://github.com/intel/onnxruntime/blob/master/onnxruntime/test/perftest/command_args_parser.cc please?

Btw is there a documentation dedicated to OVEP-specific runtime options?

Reply:
there's also the "valid_provider_keys" field (added recently, probably)
https://github.com/intel/onnxruntime/blob/89965af1bb134b6c717b942477f4a54d0976a822/onnxruntime/core/providers/openvino/contexts.h#L106
could this be added there too?